File: teste.swift
func primo(num:Int) -> Bool {
	if num < 2 {
		return false
	}
	for i in 2..<num {
		if num % i == 0 {
			return false
		}
	}
	return true
}

for i in 0..<100000 {
	if primo(num: i) {
		print(i)
	}
}

File: teste.pl
sub primo {
	$num = shift;
	if($num < 2){
		return 0;
	}
	for(my $i = 2; $i < $num; $i++){
		if($num % $i == 0){
			return 0;
		}
	}
	return 1;
}

for(my $i = 0; $i < 100000; $i++){
	if(primo($i)){
		print "$i\n";
	}
}